Build failure: biboumi

### Nixpkgs version

- Unstable (25.11)

### Steps to reproduce

build `biboumi`

### Can Hydra reproduce this build failure?

Yes, Hydra can reproduce this build failure.

### Link to Hydra build job

https://hydra.nixos.org/build/308959372

### Relevant log output

```console
> Running phase: updateAutotoolsGnuConfigScriptsPhase
       > Running phase: configurePhase
       > substituteStream() in derivation biboumi-9.0: WARNING: '--replace' is deprecated, use --replace-{fail,warn,quiet}. (file 'CMakeLists.txt')
       > cmake flags: -DCMAKE_FIND_USE_SYSTEM_PACKAGE_REGISTRY=OFF -DCMAKE_FIND_USE_PACKAGE_REGISTRY=OFF -DCMAKE_EXPORT_NO_PACKAGE_REGISTRY=ON -DCMAKE_BUILD_TYPE=Release -DCMAKE_INSTALL_LOCALEDIR=/nix/store/qffwvzvcqnb1dd3sx9mfxr7i7x5ypn4j-biboumi-9.0/share/locale -DCMAKE_INSTALL_LIBEXECDIR=/nix/store/qffwvzvcqnb1dd3sx9mfxr7i7x5ypn4j-biboumi-9.0/libexec -DCMAKE_INSTALL_LIBDIR=/nix/store/qffwvzvcqnb1dd3sx9mfxr7i7x5ypn4j-biboumi-9.0/lib -DCMAKE_INSTALL_DOCDIR=/nix/store/qffwvzvcqnb1dd3sx9mfxr7i7x5ypn4j-biboumi-9.0/share/doc/biboumi -DCMAKE_INSTALL_INFODIR=/nix/store/qffwvzvcqnb1dd3sx9mfxr7i7x5ypn4j-biboumi-9.0/share/info -DCMAKE_INSTALL_MANDIR=/nix/store/qffwvzvcqnb1dd3sx9mfxr7i7x5ypn4j-biboumi-9.0/share/man -DCMAKE_INSTALL_INCLUDEDIR=/nix/store/qffwvzvcqnb1dd3sx9mfxr7i7x5ypn4j-biboumi-9.0/include -DCMAKE_INSTALL_SBINDIR=/nix/store/qffwvzvcqnb1dd3sx9mfxr7i7x5ypn4j-biboumi-9.0/sbin -DCMAKE_INSTALL_BINDIR=/nix/store/qffwvzvcqnb1dd3sx9mfxr7i7x5ypn4j-biboumi-9.0/bin -DCMAKE_INSTALL_NAME_DIR=/nix/store/qffwvzvcqnb1dd3sx9mfxr7i7x5ypn4j-biboumi-9.0/lib -DCMAKE_POLICY_DEFAULT_CMP0025=NEW -DCMAKE_FIND_FRAMEWORK=LAST -DCMAKE_STRIP=/nix/store/dmypp1h4ldn0vfk3fi6yfyf5yxp9yz0k-gcc-wrapper-14.3.0/bin/strip -DCMAKE_RANLIB=/nix/store/dmypp1h4ldn0vfk3fi6yfyf5yxp9yz0k-gcc-wrapper-14.3.0/bin/ranlib -DCMAKE_AR=/nix/store/dmypp1h4ldn0vfk3fi6yfyf5yxp9yz0k-gcc-wrapper-14.3.0/bin/ar -DCMAKE_C_COMPILER=gcc -DCMAKE_CXX_COMPILER=g++ -DCMAKE_INSTALL_PREFIX=/nix/store/qffwvzvcqnb1dd3sx9mfxr7i7x5ypn4j-biboumi-9.0
       > CMake Error at CMakeLists.txt:1 (cmake_minimum_required):
       >   Compatibility with CMake < 3.5 has been removed from CMake.
       >
       >   Update the VERSION argument <min> value.  Or, use the <min>...<max> syntax
       >   to tell CMake that the project requires at least <min> but has been updated
       >   to work with policies introduced by <max> or earlier.
       >
       >   Or, add -DCMAKE_POLICY_VERSION_MINIMUM=3.5 to try configuring anyway.
       >
       > 
       > -- Configuring incomplete, errors occurred!
```

### Additional context

https://github.com/NixOS/nixpkgs/issues/445447

### System metadata

```console
$ nix-shell -p nix-info --run "nix-info -m"
 - system: `"x86_64-linux"`
 - host os: `Linux 6.12.49, NixOS, 25.11 (Xantusia), 25.11.20251002.7df7ff7`
 - multi-user?: `yes`
 - sandbox: `yes`
 - version: `nix-env (Nix) 2.28.5`
 - nixpkgs: `/nix/store/9v6qa656sq3xc58vkxslqy646p0ajj61-source`
```

### Notify maintainers

@woffs

---

**Note for maintainers:** Please tag this issue in your pull request description. (i.e. `Resolves #ISSUE`.)


### I assert that this issue is relevant for Nixpkgs

- [x] I assert that this is a bug and not a support request.
- [x] I assert that this is not a [duplicate of an existing issue](https://github.com/NixOS/nixpkgs/issues?q=is%3Aopen+is%3Aissue+label%3A%220.kind%3A+build+failure%22). 
- [x] I assert that I have read the [NixOS Code of Conduct](https://github.com/NixOS/.github/blob/master/CODE_OF_CONDUCT.md) and agree to abide by it.

### Is this issue important to you?

Add a :+1: [reaction] to [issues you find important].

[reaction]: https://github.blog/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/
[issues you find important]: https://github.com/NixOS/nixpkgs/issues?q=is%3Aissue+is%3Aopen+sort%3Areactions-%2B1-desc

[^1]

[^1]: Please consider [giving up MS GitHub](https://sfconservancy.org/GiveUpGitHub/) or offering a non-proprietary, non-US-corporate-controlled mirror for this free software project. I wish to delete this Microsoft account in the future, but I need more projects like this to support alternative methods to send patches & contribute.

--- a/pkgs/by-name/bi/biboumi/package.nix
+++ b/pkgs/by-name/bi/biboumi/package.nix
@@ -79,6 +79,12 @@ stdenv.mkDerivation {
     "man"
   ];
 
+  cmakeFlags = [
+    # Fix breakage with CMake 4
+    "-DCMAKE_SKIP_RPATH=ON"
+    "-DCMAKE_POLICY_VERSION_MINIMUM=3.5"
+  ];
+
   preConfigure = ''
     substituteInPlace CMakeLists.txt --replace /etc/biboumi $out/etc/biboumi
     cp ${catch}/single_include/catch.hpp tests/

